src/api/Feed.js

```diff
@@ -1369,6 +1519,77 @@ class Feed extends Base {
         });
     };
 
+    /**
+     * Create a reply to annotation or comment, and make a pending item to be replaced once the API is successful.
+     *
+     * @param {BoxItem} file - The file to which the task is assigned
+     * @param {Object} currentUser - the user who performed the action
+     * @param {string} parentId - id of the parent annotation
+     * @param {CommentFeedItemType} parentType - type of the parent item
+     * @param {string} text - the comment text
+     * @param {boolean} hasMention - true if there is an @mention in the text
+     * @param {Function} successCallback - the success callback
+     * @param {Function} errorCallback - the error callback
+     * @return {void}
+     */
+    createReply(
+        file: BoxItem,
+        currentUser: User,
+        parentId: string,
+        parentType: CommentFeedItemType,
+        text: string,
+        hasMention: boolean,
+        successCallback: Function,
+        errorCallback: ErrorCallback,
+    ): void {
+        const { id, permissions } = file;
+        if (!id || !permissions) {
+            throw getBadItemError();
+        }
+
+        const uuid = uniqueId('comment_');
+        const commentData = {
+            id: uuid,
+            tagged_message: text,
+            type: 'comment',
+        };
+
+        this.file = file;
+        this.errorCallback = errorCallback;
+        this.addPendingReply(parentId, currentUser, commentData);
+
+        const successCallbackFn = (comment: Comment) => {
+            this.createReplySuccessCallback(comment, parentId, uuid, successCallback);
+        };
+        const errorCallbackFn = (e: ErrorResponseData, code: string) => {
```

Comment:
Is there a better name for this instead of `e`? Perhaps something a bit more descriptive?

Reply:
I've renamed it to `error`.